app/src/main/res/values-zh-rTW/strings.xml

```diff
@@ -453,14 +467,22 @@
     <string name="settings_privacy">隱私</string>
     <string name="settings_privacy_secure">不允許螢幕擷取</string>
     <string name="settings_privacy_secure_summary">啟用後，將無法對本程式進行螢幕擷取，同時，將不會在系統任務切換器中顯示該程式的預覽內容，重新啟動本程式以套用此變更</string>
+    <string name="clear_search_history">清除裝置上的搜尋記錄</string>
+    <string name="clear_search_history_summary">移除曾在此裝置上進行過的搜尋</string>
+    <string name="clear_search_history_confirm">清除搜尋記錄？</string>
+    <string name="search_history_cleared">已清除搜尋記錄</string>
     <string name="download_service">下載服務</string>
     <string name="favorite_name">收藏圖庫</string>
+    <string name="dark_theme">深色主題</string>
```

Comment:
Dark theme is already translated here. https://github.com/FooIbar/EhViewer/blob/e2f024ae2917c11198172bb58e63782b82e3dfd2/app/src/main/res/values-zh-rTW/strings.xml#L596

Reply:
Sorry, my oversight. What should I do now? I'm not familiar with this